testproject/Assets/InitTestScene637509123545456028.unity

```diff
@@ -0,0 +1,452 @@
+%YAML 1.1
```

Comment:
this scene should be an artifact from a test run, I wonder how it ended being here o.o